pkg/version/version_test.go

```diff
@@ -0,0 +1,78 @@
+package version
+
+import (
+	"testing"
+)
+
+func TestInfo_String(t *testing.T) {
+	type fields struct {
+		GitVersion   string
+		GitCommit    string
+		GitTreeState string
+		BuildDate    string
+		GoVersion    string
+		Compiler     string
+		Platform     string
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   string
+	}{
+		{
+			name: "test1",
+			fields: fields{
+				GitVersion:   "1.8.3.1",
+				GitCommit:    "da070e68f3318410c8c70ed8186a2bc4736dacbd",
+				GitTreeState: "clean",
+				BuildDate:    "2022-08-31T13:09:22Z",
+				GoVersion:    "go1.18.3",
+				Compiler:     "gc",
+				Platform:     "linux/amd64",
+			},
+			want: "version.Info{GitVersion:\"1.8.3.1\", GitCommit:\"da070e68f3318410c8c70ed8186a2bc4736dacbd\", GitTreeState:\"clean\", BuildDate:\"2022-08-31T13:09:22Z\", GoVersion:\"go1.18.3\", Compiler:\"gc\", Platform:\"linux/amd64\"}",
+		},
+		{
+			name: "test2",
+			fields: fields{
+				GitVersion:   "v1.1.1-6-gf20c721a",
+				GitCommit:    "da070e68f3318410c8c70ed8186a2bc4736dacbd",
+				GitTreeState: "clean",
+				BuildDate:    "2022-08-31T13:09:22Z",
+				GoVersion:    "go1.18.3",
+				Compiler:     "gc",
+				Platform:     "linux/amd64",
+			},
+			want: "version.Info{GitVersion:\"v1.1.1-6-gf20c721a\", GitCommit:\"da070e68f3318410c8c70ed8186a2bc4736dacbd\", GitTreeState:\"clean\", BuildDate:\"2022-08-31T13:09:22Z\", GoVersion:\"go1.18.3\", Compiler:\"gc\", Platform:\"linux/amd64\"}",
+		},
+		{
+			name: "test3",
+			fields: fields{
+				GitVersion:   "v1.1.1-6-gf20c721a",
+				GitCommit:    "5ee9f6776e1225fa127f882ed84ac9d75a38730a",
+				GitTreeState: "clean",
+				BuildDate:    "2022-08-31T13:09:22Z",
+				GoVersion:    "go1.18.3",
+				Compiler:     "gc",
+				Platform:     "linux/aarch64",
+			},
+			want: "version.Info{GitVersion:\"v1.1.1-6-gf20c721a\", GitCommit:\"5ee9f6776e1225fa127f882ed84ac9d75a38730a\", GitTreeState:\"clean\", BuildDate:\"2022-08-31T13:09:22Z\", GoVersion:\"go1.18.3\", Compiler:\"gc\", Platform:\"linux/aarch64\"}",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			info := Info{
+				GitVersion:   tt.fields.GitVersion,
+				GitCommit:    tt.fields.GitCommit,
+				GitTreeState: tt.fields.GitTreeState,
+				BuildDate:    tt.fields.BuildDate,
+				GoVersion:    tt.fields.GoVersion,
+				Compiler:     tt.fields.Compiler,
+				Platform:     tt.fields.Platform,
+			}
+			if got := info.String(); got != tt.want {
```

Comment:
For string function, I think one case is enough. BTW, would u mind helping add ut for `Get` function.